pkg/getty/etc/profile.d/namespace.sh

```diff
@@ -1 +1,23 @@
-export PS1="(ns: getty) $PS1"
+#/bin/sh
+
+in_a_ns() {
+	IN_A_NS=0
+	NOT_IN_A_NS=1
+
```

Comment:
I would add if `readlink /proc/self/ns/mnt` and `readlink /proc/1/ns/mnt` we are in a namespace, this works well with the current setup of shared pid namespace but not mount. (It doesnt work if you are in a pid namespace).